### ISSUE TITLE
Can we do multiprocessing or distributed computing through PyFlink?

### ISSUE THREAD

**Issue by abhalawat** [pyflink/playgrounds#21]
I am trying to get about 14million data and want this process to work faster.Is there any way PyFlink could help?

**Comment by dianfu** [comment]
@abhalawat It's processing the data in a distributed way. This is the ability of Flink runtime.

**Comment by abhalawat** [comment]
How will I achieve  that?

**Comment by dianfu** [comment]
Yes, when you submit a PyFlink to a remote cluster, e.g. YARN, K8s, etc, it will execute it in a distributed manner (you need to set the parallelism to a value other than 1). See https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/deployment/cli/#submitting-pyflink-jobs for more details on how to submit PyFlink jobs to a remote cluster.

**Comment by abhalawat** [comment]
I followed this doc and created task manager and job manager.But while running the first code i.e. 1_word_count.py.

I ma facing the error as:
![flink](https://user-images.githubusercontent.com/97433422/155269024-0ee67699-0c0a-4f53-a4cd-2cd26a52fc2f.png)

**Comment by dianfu** [comment]
Could you post the full exception stack?

**Comment by abhalawat** [comment]
Sure here it is:
![error1](https://user-images.githubusercontent.com/97433422/155465609-1d266c42-23d6-422d-849c-f444840d50b1.png)
![error2](https://user-images.githubusercontent.com/97433422/155465614-0bb8d8b9-75dc-488d-92cb-98652e2c32c7.png)
![error3](https://user-images.githubusercontent.com/97433422/155465617-0061a3a8-c57e-47ce-bd4b-581b9d9a1d51.png)
![error4](https://user-images.githubusercontent.com/97433422/155465623-762f2cd5-ecb3-41a0-8a7b-fc1dd8f175b0.png)

**Comment by dianfu** [comment]
@abhalawat From the exception stack, it failed to create the output directory. There should be some permission issues. You could change the sink connector to other connectors such as `[print](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/table/print/)` to work around this problem.

**Comment by abhalawat** [comment]
Should I change this command:
```
t_env.execute_sql("""
         CREATE TABLE mySink (
           word STRING,
           `count` BIGINT
         ) WITH (
           'connector' = 'filesystem',
           'format' = 'csv',
           'path' = '/opt/examples/table/output/word_count_output'
         )
     """)
```

**Comment by abhalawat** [comment]
I created new folder for output named as result then too it was showing error:

![output1](https://user-images.githubusercontent.com/97433422/155468294-65d38e97-e8a4-4bc2-a962-2e6837aba507.png)
![output2](https://user-images.githubusercontent.com/97433422/155468302-baf3fc21-7cca-4250-96fd-e544943954cb.png)
![output3](https://user-images.githubusercontent.com/97433422/155468306-b5a602ca-6735-42a6-a678-f4a696c68c94.png)
![output4](https://user-images.githubusercontent.com/97433422/155468309-e465b420-c912-495f-bb83-610ef05c695f.png)

**Comment by abhalawat** [comment]
Also,In here connector has to be file system because I am attaching file in SQL query.
